src/main/java/com/mg105/utils/ItemConstants.java

```diff
@@ -1,17 +1,23 @@
 package com.mg105.utils;
 
 
+import com.mg105.entities.items.HealthPotion;
+
 public class ItemConstants {
 
     public static final String UPGRADE_TOKEN_NAME = "Upgrade Token";
-    public static final String UPGRADE_TOKEN_DESCRIPTION = "One use increases a " + "specific stat on a specific character by one.";
+    public static final String UPGRADE_TOKEN_DESCRIPTION = "One use increases a " +
+        String.valueOf(HealthPotion.getHealingPoints()) + "specific stat on a specific character by one.";
 
     public static final String HEALTH_POTION_NAME = "Health Potion";
-    public static final String HEALTH_POTION_DESCRIPTION = "This potion will heal a character by a certain amount";
+    public static final String HEALTH_POTION_DESCRIPTION = "This potion will heal" +
+        String.valueOf(HealthPotion.getHealingPoints()) + "points to a character";
```

Comment:
I would phrase this more like "This potion will restore ___ health points to a character", but it's not important